dlt/_workspace/cli/utils.py

```diff
@@ -0,0 +1,118 @@
+import os
+import shutil
+from typing import Any
+
+import dlt
+from dlt._workspace.profile import LOCAL_PROFILES
+from dlt.cli.exceptions import CliCommandException
+from dlt.common.configuration.specs.pluggable_run_context import (
+    RunContextBase,
+    ProfilesRunContext,
+)
+from dlt.common.storages.file_storage import FileStorage
+
+from dlt.cli import echo as fmt
+
+
+def display_run_context_info() -> None:
```

Comment:
Ok, this is where this profile output comes from. I think we need this to contain info for non-pinned default profiles too.